.travis.yml

```diff
@@ -63,7 +63,9 @@ jobs:
       before_install:
         - if [ -z "${TRAVIS_TAG}" ]; then RELEASE_VERSION="SNAPSHOT-${TRAVIS_BUILD_NUMBER}"; else RELEASE_VERSION="${TRAVIS_TAG}";fi
         - echo "Setting version to $RELEASE_VERSION"
-      install: ./gradlew -PbuildVersion=$RELEASE_VERSION assemble packageDocumentation
+        - if [ -z "${TRAVIS_TAG}" ]; then echo "Not release build - skipping docker login."; else ./resources/docker-login.sh;fi
```

Comment:
Can you just do the login here?

``... else docker login -u "$DOCKER_HUB_USER" -p "$DOCKER_HUB_PW";fi ``